jbdevstudio/com.jboss.jbds.central.discovery/plugin.xml

```diff
@@ -43,6 +43,33 @@ reasonable, reporting issues to these providers as required.</description>
 
       <!-- use connectorCategory relevance value to sort from largest (80) to smallest (20) -->
 
+      <connectorCategory
+            description="Patches"
+            id="org.jboss.tools.central.discovery.a.a.patches"
+            name="Patches"
+            relevance="90">
+         <group id="org.jboss.tools.central.discovery.a.a.patches" /> 
+      </connectorCategory>
+
+      <connectorDescriptor
+            categoryId="org.jboss.tools.central.discovery.a.a.patches"
+            groupId="org.jboss.tools.central.discovery.a.a.patches"
+            description="Web Tools 3.6.3 Patch Feature"
```

Comment:
Sure, we can use whatever you want there. I can also mention JBDS-3383 or the bugzilla(s), but we can't LINK to them so users would have to manually re-type (or maybe select the text & copy it) to follow the URL.

Could you be more specific about WHICH javascript projects? Do you want a link to JIRA or bugzilla?